docs/encyclopedia/temporal-nexus-service-operations.mdx

```diff
@@ -0,0 +1,146 @@
+
+Calling a Nexus `Operation` from a caller `Workflow` presents as a non-blocking, arbitrary duration operation. This schedules the `Operation` with the caller’s Temporal cluster and does not block caller `Workflow` execution. The caller’s Nexus Machinery is responsible for making the underlying Nexus RPC calls and updating the caller’s Workflow history with the result.
+
+#### Operation Lifecycle
+
+The Temporal Go SDK is used for the operation lifecycle examples below.
+
+Handlers are created with the Go SDK using:
+
+- `temporalnexus.NewWorkflowRunOperation()`
+  - start a `Workflow` as an asynchronous operation
+- `temporalnexus.NewSyncOperation()`
+  - invoke an underlying `Query` or `Signal` as a synchronous operation
+  - invoke an `Update` as a synchronous operation
+  - execute arbitrary code as a synchronous operation
+
+Caller Workflows use the following to execute a Nexus Operation:
+
+- `svc := Workflow.CreateNexusClient(endpoint, service)` and then
+- `svc.ExecuteOperation()`
+
+##### Asynchronous Operation Lifecycle {#asynchronous-operation-lifecycle}
+
+An asynchronous Nexus Operation may take up to 2 months to complete, to allow for callback URL token validation with asymmetric key rotation at those intervals.
+
+The lifecycle of an asynchronous Nexus `Operation`:
+
+1. Caller’s Workflow does a `myServiceClient.ExecuteOperation()`
+   1. **`NexusOperationScheduled`** added to caller’s event history
+   2. Nexus `Operation` added to Nexus Machinery in the caller’s Temporal cluster
+   3. Immediately returns a Nexus `OperationHandle`, so the Workflow doesn’t block.
+   4. Caller’s Workflow can interact with the `Operation` using the `OperationHandle`
+      1. `WaitStarted()`, `GetResult()`, and the ability to cancel.
+2. Start Nexus operation on the target cluster is attempted by the caller’s Temporal cluster
+   1. Via the Nexus Machinery in the target cluster
+   2. Retries are processed by the Nexus Machinery in the caller’s Temporal cluster and tracked as part of the caller Workflow, similar to Activities.
+3. Nexus `Operation handler` in the target cluster processes the Nexus `Operation`
+   1. `Operation handler` pulls operation `Task` from Task Queue
+   2. `Operation handler` starts a `Workflow` in the target cluster with attached Nexus `Callback`
+   3. Nexus operation `id` is returned since the Nexus operation is asynchronous
+   4. **`NexusOperationStarted`** added to caller’s event history
+   5. Operation `id` is added to the caller’s operation `OperationHandle`
+4. Handler’s Nexus Machinery sends a Nexus completion `Callback` when the `Workflow` has completed.
+   1. Operation `result` is returned via callback since the operation is asynchronous
+   2. **`NexusOperationCompleted`** added to caller’s event history
+5. Caller’s Workflow `handle.GetResult()` returns the result
+
+##### Synchronous Operations Lifecycle
+
+An synchronous Nexus Operation may take up to 10 seconds to complete and will then timeout.
```

Comment:
I'm correcting this but this statement is confusing IMHO, it's not the operation that can take up to 10 seconds, it's a single attempt to start the operation.

```suggestion
A synchronous Nexus Operation may take up to 10 seconds to complete and will then timeout.
```